conformance/base/manifests.yaml

```diff
@@ -350,3 +350,20 @@ spec:
         resources:
           requests:
             cpu: 10m
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: secrets
+  labels:
+    gateway-conformance: secrets
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: certificate
+  namespace: secrets
+type: kubernetes.io/tls
+data:
+  tls.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUU2RENDQXRBQ0NRQ2xDM0tMdkJiSlBqQU5CZ2txaGtpRzl3MEJBUXNGQURBMk1Rc3dDUVlEVlFRR0V3SlYKVXpFVE1CRUdBMVVFQ2d3S1MzVmlaWEp1WlhSbGN6RVNNQkFHQTFVRUF3d0piRzlqWVd4b2IzTjBNQjRYRFRJeQpNRFF4T1RFMk5ERXhOMW9YRFRJek1EUXhPVEUyTkRFeE4xb3dOakVMTUFrR0ExVUVCaE1DVlZNeEV6QVJCZ05WCkJBb01Da3QxWW1WeWJtVjBaWE14RWpBUUJnTlZCQU1NQ1d4dlkyRnNhRzl6ZERDQ0FpSXdEUVlKS29aSWh2Y04KQVFFQkJRQURnZ0lQQURDQ0Fnb0NnZ0lCQUxZRjVMT2pzNlhLMEhWUWxwamh3YXFMS016dW5FTUs5eUxISDcxUgp0dDVWOC9iUzVrTEE5NzhDeWhmVExVRHdKTy9zRXIyVHZPRE1oNG5KZWxyWGp2VEpQSzZvVlM2bWZYcXRWOVpPCk9pcDJtdE5odnBaRTloYTMralVuS1A1R2paSEFlODcxR3g3ZGZ0ajk1YTYzNUNyZEtRSEVKZWh5K2xJT3BpZjgKSGNFNmZsSWViaE9peW43bHE5cHptREZmOTZVNUhKU05sZ2ttK3lIcVd6ZE1FWlBnRXhQK3Q2Ym1HcUJHZnJpbwpITGpRRVRaSEFweFNkSkoxSzN4RWgzcmpFeitJTkhMYTdIeDBUTFRNandsR1ZJTGMza0tqNVJxb05FWTNwWUYxCmFnV2xpSy8xVzlkOFF1VlBvajNFU3E3d1gxTG00NU90NXRKTUtqbnJCOGxPVVJmV3YyQlBMYWdoRkl2WmVjNksKQ21jc25hd0hFZzE0RGJxQUdmNkNGWjdEMkoyeXRqUHArWVVrUTRQZEg1MzlkYzhVSlFuUHZXLytYRUJYMUg1dQozMmV6OUlwVVdQNHRHREpBMWVUa0orQW03VDdrVE1lbFk3ekE5RkRVOWpncWFDc21QNUIzeExCWktkN0ZadXBrCmhrbTJzWHZhazFtRzY1WkRQUC9qc2g1VExTazhUZTZJR0lOdWEzd1ZNc09icWlxRW05ZmVsWFlBOG1aUGMwRlUKRVNWSXhNeFNoOEdQZWV1dHNlTVVNb2tXTmFqNEZSU2JRR1V1bWpSZHNoV0MrV3ZIQU9zV2pRSTJhbFA5Y3cxYgpDS05CbXpiRURKLytZalJ1bXZGYUo3TmtzUDZVRklLVXdJS1VmOWdOOWs5UXFpQlU1TWIxQ1lmZWJteEJtN3oyCjQ0b2JBZ01CQUFFd0RRWUpLb1pJaHZjTkFRRUxCUUFEZ2dJQkFJZS9iRzZZcVBIYmdHbG5WMUpTckpTcERVSVcKL0REdFhuclFudzFZYkhrV3J0VWwyczdicTZobSthNndobGNqOXJWTUhwZ0JDUEE2bHY5OWprbXEyL2hsNk5EeQp0ZFNmVGVmNWhtcjJOTG1xVzhRQUQrQm5rTjhpQ3JnV1krSHpHMW5PQWhaVGFINE93azgvVmZUYmFPMWhzVStQCjFnb3NyU3Q1Zk5SSGErQ3RGWWVPZHJSUFRQMUh5TDZDdFppV2R3THA3Y0dDWXZxY28ySjZOMzQ3ank3Nmx5dzUKNFc1Zm02RW5RaXFzZGo1VitYWmt6L3k4Nnp2ZUpPLzZMN1VmeE9BR09KclFETFNXLzUzSEFYcTN4RWhYbnd2aQpweWhISkpWVmxPclZWcEdxSlRlUDVya0VXOWJYNGtqdkZoQklvZXQyKy9leTQvdU5aanREdGtVWVlrZHAwaWdKClVabjlEeGV0ZzRxZEN1aDdCcFphdmJvTWJRNGdlTXNQckFRVXBrcVJmeVh3OGFNTXV0bk0zalV1dWpSN0FiWngKc2puYWJQOTdWY0MxRTdhenkraEJTM2pjTHlyTmdzdWd6REhXb1QyeHhSNXo3NEdSMFozSGdSQlRJWnRmYTJ4RwpZeDdpeWI4ZnFTMGZrQ3YyL1hxTVh0NisvUGdabGdhaDBBSXRoSlVkOGtoOE1OZmRwZURyQVNIQWF2eEFFTzNqCjMrWTZwZzVpVWg2OEE0SVlMOHNsSlNTZlE0NzZuaGpyekp6NU5DSHUzVC9yMVEwaVZqTGNjdEFrc1NWdTNlSHcKdzB1YTVKeG5pTDUzYmwxakVPTEV0QzFVS1RNeXR5Uy94M1I2UTNoSUlNR0dyNE5nV1VGdU9aRnJURWxkODNiVwpIa3JJS0JZTlY1Zy9nWmJnCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
```

Comment:
This is a self-signed throwaway cert; however, it might trigger scanners somewhere. Any conventions for this that I don't know about?

Reply:
I think we may want to create this in the Go test definition instead of a yaml manifest. Maybe we could have a function similar to what was used in ingress-controller-conformance? (https://github.com/kubernetes-sigs/ingress-controller-conformance/blob/cec7cf25b0291933426e256065ef4afc981ea1fa/test/kubernetes/kubernetes.go#L207-L244)

Reply:
This turned out to be a good approach, thanks @robscott ! I did need to add a hook to install in-memory objects during `TestSuite.Setup` and have them clean up alongside the yaml manifests that we use for base resources today